coalib/parsing/LineParser.py

```diff
@@ -78,6 +79,12 @@ def _parse(self, line):
                      [(section_override, key), ...], value, to_append (True if
                      append delimiter is found else False), comment
         """
+        if (re.match('[^ ]#', line) or re.match('#[^ ]', line))\
+           and len(line) > 2:
+            logging.warning('This comment does not have whitespace' +
```

Comment:
Warning should contain a way to silence it. So tell the user to put it or give a fixed representation of the line in the warning. I think a correct representation should be easy in this case...